appveyor.yml

```diff
@@ -1,6 +1,8 @@
 environment:
   matrix:
-  - julia_version: 1
+  - julia_version: 1.1
+  - julia_version: 1.2
+  - julia_version: 1.3
```

Comment:
Should be enough to run on 1.3 and nightly, we will not ship this version of Pkg with older Julia anyway.